newIDE/app/src/MainFrame/index.js

```diff
@@ -959,6 +975,23 @@ const MainFrame = (props: Props) => {
     autosaveProjectIfNeeded();
   };
 
+  useGlobalCommand('LAUNCH_PREVIEW', {
```

Comment:
some thoughts on where to place commands in component code: might be a good idea to place them at single place, before the `return <blabla>`.

In a way, these hooks are "rendering" things. It's not DOM or components, they are rendering commands :) 
I suggest them to be after the logic hooks, so that a component is:

function MyComponent(props) {
  //Logic and hooks like useState()

  // useGlobalCommand hooks (the "commands" that are a bit part of the UI when you think about it)
  // return <Dialog>...</Dialog> (the UI shown on screen)
}

Reply:
Yeah, I was thinking about that actually! Will do that, sure.